extensions/binding/org.eclipse.smarthome.binding.tradfri/ESH-INF/thing/thing-types.xml

```diff
@@ -69,6 +69,26 @@
             </parameter>
         </config-description>
     </thing-type>
+    <thing-type id="group">
```

Comment:
Confirmed. Group brightness is a write-only variable. Setting it causes all lights to update it's brightness. After this it stays at this value even though we set all lights at another brightness value.

Don't know if we should include it if it has such strange behaviour. What do you think?